### PR TITLE
[ADD] calendar usability

### PR DESCRIPTION
**Pull request by bealdav**
capture du 2016-01-05 18-59-18

### CODE REVIEW

File: calendar_usability/resource.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        return {
+            'hour_from': hour_from,
+            'hour_to': hour_to,
+            'name': '.',
+            'dayofweek': str(day),
+        }
+
+    @api.model
+    def create(self, vals):
+        if not vals.get('attendance_ids'):
```
Comment (sebastienbeau) [inline calendar_usability/resource.py]: Best to use default value

File: calendar_usability/resource.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            hour_from2=13,
+            hour_to2=17,
+        )
+
+    @api.model
+    def _populate_attendance(self, day, hour_from, hour_to):
+        return {
+            'hour_from': hour_from,
+            'hour_to': hour_to,
+            'name': '.',
```
Comment (sebastienbeau) [inline calendar_usability/resource.py]: maybe it will be better to put the day in the name and hide it in the view?

**Comment by bealdav** [comment]
@sebastienbeau , if ok for you could you merge ?

File: calendar_default_value/resource.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        for rec in self:
+            rec.display_name = "%s: %s" % (rec.name, rec.hour_range)
+
+    @api.model
+    def default_get(self, fields_list):
+        "'attendance_ids' field default value"
+        values = super(ResourceCalendar, self).default_get(fields_list)
+        vals = []
+        params = self.get_my_calendar_data()
+        for hours in range(0, params.endday):
```
Comment (sebastienbeau) [inline calendar_default_value/resource.py]: why did you call this variable "hours"? "day" is maybe a better name?

Comment (bealdav) [inline calendar_default_value/resource.py]: It was 'day' before, but you can have several hour range a day.

**Pull request merged**